tests/expected/loop-contract/loop_with_previous.expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESSFUL
```

Comment:
This file has the wrong name and is thereby causing tests to fail - should be "loop_with_prev.expected"